packages/excalidraw/components/App.tsx

```diff
@@ -7747,6 +7759,17 @@ class App extends React.Component<AppProps, AppState> {
             mutateElement(newElement, {
               points: [...points, [dx, dy]],
             });
+          } else if (points.length > 1 && isElbowArrow(newElement)) {
+            mutateElbowArrow(
+              newElement,
+              this.scene,
+              [...points.slice(0, -1), [dx, dy]],
+              [0, 0],
+              undefined,
+              {
+                isDragging: true,
+              },
+            );
```

Comment:
this was removed by mistake.

Case: updating elbow arrow points as you create it.

Reply:
a test would be nice. Maybe just checking if points are middle points are being set/updated when creating "2-point" elbow arrow?